tests/common/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library", "cc_test")
+
+cc_test(
+    name = "utils_unit_test",
```

Comment:
I think `utils_test` is enough? I don't see any other tests we will have for utils.

Reply:
Renamed